thirdparty/openapi3/schemas/ConsentRequestsIDPutResponseOTP.yaml

```diff
@@ -0,0 +1,37 @@
+title: ConsentRequestsIDPutResponseOTP
+type: object
+description: |
+  The object sent in a `PUT /consentRequests/{ID}` request.
+
+  Schema used in the request consent phase of the account linking OTP/SMS flow.
+properties:
+  id:
+    $ref: '../../../v1.0/openapi3/schemas/CorrelationId.yaml'
+  initiatorId:
+    type: string
+    description: The id of the PISP who will initiate transactions on a user's behalf.
+  scopes:
+    type: array
+    items:
+      $ref: './Scope.yaml'
+  authChannels:
+    type: array
+    items:
+      $ref: './AuthorizationChannelType.yaml'
+    # TODO: min and max
+  callbackUri:
+    # TODO: enforce uri type
+    type: string
+    description: The callback uri that the user will be redirected to after completing the WEB auth channel.
+  authUri:
```

Comment:
Can we either drop this, or ensure it's set to `null`?

Reply:
Don't believe we can ensure it's null in OA3. `nullable` is there only option since there is no `null` type.
We could just drop it. Since it's always null.

Reply:
If we drop it. Might need to check the services that are expecting it to be null.

Reply:
I can't ensure it will be null so...I'll drop it. Hope that doesn't cause any trouble.

Reply:
Yeah I think that's better anyway.